src/discussions/common/HoverCard.jsx

```diff
@@ -12,7 +12,7 @@ import classNames from 'classnames';
 import { useIntl } from '@edx/frontend-platform/i18n';
 
 import { ThreadType } from '../../data/constants';
-import { useUserPostingEnabled } from '../data/hooks';
+import { useHasLikePermission, useUserPostingEnabled } from '../data/hooks';
```

Comment:
i think the hook name should be useUserLikePermission or useUserHasLikePermission